apps/expo/src/app/index.tsx

```diff
@@ -101,11 +113,11 @@ const Index = () => {
   });
 
   return (
-    <SafeAreaView className="bg-[#1F104A]">
+    <SafeAreaView style={{ backgroundColor: "#1F104A" }}>
```

Comment:
@marklawlor any idea why certain elements aren't getting classes applied? I know v4 is still early and experimental but should this work?

Reply:
This will be in the next patch, but as a temporary workaround you can add this to your application.

```
const { SafeAreaView } = require("react-native-safe-area-context");
cssInterop(SafeAreaView, { className: "style" });

remapProps(TouchableOpacity, { className: "style" });
remapProps(TouchableHighlight, { className: "style" });
remapProps(TouchableWithoutFeedback, { className: "style" });
```

As to why it only works on some components, NativeWind v4 tries to only operate on the leaf nodes of your render tree (the native components). So you need to enable the interop on your native components. For the Touchables, they don't pass down all their props internally, so we need to remap their props.

TLDR: I forgot to tag the Touchables and SafeAreaView is a native component

I've added these changes in https://github.com/marklawlor/nativewind/commit/546606209a422c223065858b3335f3102951176e & https://github.com/marklawlor/nativewind/commit/2b15c2e831813ef836e66278a50bb270d7e529bc, so you can remove the workaround once they are published.

FYI: Your should consider updating the `Touchable*` for `Pressable`, as `Touchable*` are effective deprecated and no longer receive updates like the new Pointer API.

Reply:
Awesome, thanks!

Reply:
Do you think it would be possible to add modifiers like `pressed:` for doing something like this:

```ts
<Pressable
  // this 
  className="bg-blue-500 pressed:opacity-50"
 
  // instead of this
  style={({ pressed }) => ({ opacity: pressed ? 0.5 : 1, backgroundColor: "..." })}
>
  Click Me
</Pressable>
```

Reply:
You can use the `active:` modifier for that. `active:opacity-50`


If you use the `active:` modifier on a `<View />`, NativeWind will automatically upgrade it to a `<Pressable />` for you. So you can simply write `<View className="bg-blue-500 active:opacity-50" />`.

Also, `<TouchableOpacity />` used to be animated, so you may want to mimic that behaviour.

You can write `<View className="bg-blue-500 active:opacity-50 transition" />`

NativeWind will then convert it to be an `Animated.Pressable` and animate the `opacity` on press.